src/app/modules/main/chat_section/chat_content/module.nim

```diff
@@ -396,6 +396,7 @@ method onCommunityChannelEdited*(self: Module, chatDto: ChatDto) =
   self.view.chatDetails.setCanView(chatDto.canView)
   self.view.chatDetails.setCanPostReactions(chatDto.canPostReactions)
   self.view.chatDetails.setHideIfPermissionsNotMet(chatDto.hideIfPermissionsNotMet)
+  self.view.chatDetails.setName(chatDto.name)
```

Comment:
I realize I misread the conditon in `onChatUpdated` and I thought it said if `type == OnToOne` but it's the opposite.

So it means we're also missing the icon change:
```nim
 self.view.chatDetails.setIcon(chatItem.icon)
```

Reply:
Added the change. I'm just curious, how do I set channel icons?

Reply:
Hmm, now I realize that community channels don't have icons :sweat_smile: 

They do have emojis, but I don't think it matters, so maybe that change doesn't matter.

Can you test to change the channel emoji with and without that change? Feel free to remove the change if it's not needed